Update README.md

earlAchromatic opened the pull request.

earlAchromatic committed:
Update README.md

earlAchromatic commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable-dev-jacob-default-rtdb.firebaseapp.com/reviews/Hyperion-Web/Nebula/8)
<!-- Reviewable:end -->

earlAchromatic commented:
+reviewer:@oscl8tr 





<!-- Sent from Reviewable.io -->

earlAchromatic commented:
*[`README.md` line 27 at r1](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-NST2GFt8iQyK9UZyXdg:-NST2GFt8iQyK9UZyXdh:b32wwo) ([raw file](https://github.com/Hyperion-Web/Nebula/blob/85a834fc01bd48df56e0ddaef9557b3f69cbb107/README.md#L27)):*
> ```Markdown
> 
> 3. Add Storybook for visual testing & documentation?
> ```

sf


<!-- Sent from Reviewable.io -->

oscl8tr commented:
*[`README.md` line 27 at r1](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-NST2GFt8iQyK9UZyXdg:-NYxkKvM0rMVsvYCV9I8:b-7sqhy0) ([raw file](https://github.com/Hyperion-Web/Nebula/blob/85a834fc01bd48df56e0ddaef9557b3f69cbb107/README.md#L27)):*
<details><summary><i>Previously, earlAchromatic (Jacob Milhorn) wrote…</i></summary><blockquote>

sf
</blockquote></details>

block


<!-- Sent from Reviewable.io -->

earlAchromatic commented:
*[`README.md` line 27 at r1](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-NST2GFt8iQyK9UZyXdg:-NYxkXiWCw4-BmU68Vdd:b-896fix) ([raw file](https://github.com/Hyperion-Web/Nebula/blob/85a834fc01bd48df56e0ddaef9557b3f69cbb107/README.md#L27)):*
<details><summary><i>Previously, oscl8tr wrote…</i></summary><blockquote>

block
</blockquote></details>

Done.


<!-- Sent from Reviewable.io -->

earlAchromatic commented:
*[`README.md` line 28 at r1](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-NYyPS0N-aSz3OXOZrNH:-NYyPS0O9ZzVRsb2lwp1:b-aakijm) ([raw file](https://github.com/Hyperion-Web/Nebula/blob/9093665033e6f5ea05f979d0a24a5b4441430575/README.md#L28)):*
> ```Markdown
> 3. Add Storybook for visual testing & documentation?
> 
> Possibly limit exposed components to assembled molecules and layouts. Abstract away what is in between the content and the complicated layouts and logic.
> ```

Typically you'd just set the `duration` to `animator.defaultOptions.duration * 2/3`, so you'd automatically trigger any `begin`/`complete` hooks without needing to duplicate them.
sdfsa
dg
asd

asdg
asdg


<!-- Sent from Reviewable.io -->

oscl8tr reviewed (changes requested):
> *[Reviewable](http://localhost:8080/reviews/Hyperion-Web/Nebula/8)* status: 0 of 1 files reviewed, 2 unresolved discussions (waiting on @earlAchromatic)

___
*[`-- commits` line 1 at r1](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-NZiCLbb6dRaV9Md2lbb):*
sg


<!-- Sent from Reviewable.io -->

earlAchromatic reviewed (commented):
Just a comment to change height.

> *[Reviewable](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-:-NZlf9Be296P8O-GQyaU:bork3e5)* status: 0 of 1 files reviewed, 2 unresolved discussions (waiting on @oscl8tr)



<!-- Sent from Reviewable.io -->

earlAchromatic committed:
Update README.md

earlAchromatic committed:
Update README.md

oscl8tr reviewed (commented):
How about this long message.

> *[Reviewable](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-:-NZlhhqy2fJbH8iGVBP1:b-ub578v)* status: 0 of 1 files reviewed, 1 unresolved discussion (waiting on @earlAchromatic)



<!-- Sent from Reviewable.io -->

earlAchromatic committed:
Update README.md

earlAchromatic committed:
Update README.md

oscl8tr reviewed (commented):
> Reviewed 1 of 1 files at r2, 1 of 1 files at r3.
> *[Reviewable](http://localhost:8080/reviews/Hyperion-Web/Nebula/8)* status: 0 of 1 files reviewed, all discussions resolved (waiting on @earlAchromatic)



<!-- Sent from Reviewable.io -->

earlAchromatic commented:
@Hyperion-Web/subs  





<!-- Sent from Reviewable.io -->

prcl8tr reviewed (approved):
> Reviewed all commit messages.
> *[Reviewable](http://localhost:8080/reviews/Hyperion-Web/Nebula/8)* status: 2. General review (waiting on @oscl8tr)



<!-- Sent from Reviewable.io -->

earlAchromatic reviewed (commented):
> Reviewed 1 of 1 files at r1, 1 of 1 files at r4.
> *[Reviewable](http://localhost:8080/reviews/Hyperion-Web/Nebula/8)* status: 2. General review (waiting on @earlAchromatic)



<!-- Sent from Reviewable.io -->

prcl8tr reviewed (commented):
> Reviewed 1 of 1 files at r1.
> *[Reviewable](http://localhost:8080/reviews/Hyperion-Web/Nebula/8)* status: 2. General review (waiting on @earlAchromatic)



<!-- Sent from Reviewable.io -->

prcl8tr reviewed (dismissed):
> Reviewed 1 of 1 files at r4.
> *[Reviewable](http://localhost:8080/reviews/Hyperion-Web/Nebula/8)* status: 3. Specialized review: needs approval from squad-of-life (waiting on @earlAchromatic)



<!-- Sent from Reviewable.io -->

earlAchromatic commented:
*[`README.md` line 24 at r1](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-Ny1e0KZDM_iXxEr4pUm:-Ny1e0KZDM_iXxEr4pUn:b2nic5s) ([raw file](https://github.com/Hyperion-Web/Nebula/blob/9093665033e6f5ea05f979d0a24a5b4441430575/README.md#L24)):*
> ```Markdown
> 1. Vue 3 chosen for most modern approach to single file components and future-proof design. In near future, mitosis can be used to port these to other frameworks. Also, Vue has built in .ce mode which allows export of all of these as custom elements (the lib should provide this as a `dist` file).
> 
> 2. CSS structure. CSS vars will be the primary way we allow customization. So all components should be styled using these by default to remain flexible. The final interface should allow dead simple variable assignment on the root element to override the `nebula` defaults.
> ```

sdf


<!-- Sent from Reviewable.io -->

earlAchromatic reviewed (commented):
> *[Reviewable](http://localhost:8080/reviews/Hyperion-Web/Nebula/8)* status: 0 of 1 files reviewed, 2 unresolved discussions (waiting on @oscl8tr and @prcl8tr)

___
*[`README.md` line 26 at r3](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-Ny1rJXz8NlREHcx5d8i:-Ny1rJXz8NlREHcx5d8j:b2nic5s) ([raw file](https://github.com/Hyperion-Web/Nebula/blob/85a834fc01bd48df56e0ddaef9557b3f69cbb107/README.md#L26)):*
> ```Markdown
> 2. CSS structure. CSS vars will be the primary way we allow customization. So all components should be styled using these by default to remain flexible. The final interface should allow dead simple variable assignment on the root element to override the `nebula` defaults.
> 
> 3. Add Storybook for visual testing & documentation?
> ```

sdf



_Code quote:_
```Markdown
2. CSS structure. CSS vars will be the primary way we allow customization. So all components should be styled using these by default to remain flexible. The final interface should allow dead simple variable assignment on the root element to override the `nebula` defaults.

3. Add Storybook for visual testing & documentation?
```


<!-- Sent from Reviewable.io -->

earlAchromatic reviewed (commented):
> Reviewed 1 of 1 files at r3.
> *[Reviewable](http://localhost:8080/reviews/Hyperion-Web/Nebula/8)* status: 0 of 1 files reviewed, 2 unresolved discussions (waiting on @oscl8tr and @prcl8tr)



<!-- Sent from Reviewable.io -->

earlAchromatic reviewed (commented):
> Reviewed 1 of 1 files at r4, all commit messages.
> *[Reviewable](http://localhost:8080/reviews/Hyperion-Web/Nebula/8)* status: all files reviewed, 2 unresolved discussions (waiting on @oscl8tr and @prcl8tr)



<!-- Sent from Reviewable.io -->

earlAchromatic reviewed (commented):
> *[Reviewable](http://localhost:8080/reviews/Hyperion-Web/Nebula/8)* status: 0 of 1 files reviewed, 2 unresolved discussions (waiting on @oscl8tr and @prcl8tr)

___
*[`README.md` line 18 at r4](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-NyvLVU0-FzBiLxWcHGn:-NyvLVU0-FzBiLxWcHGo:b32wwo) ([raw file](https://github.com/Hyperion-Web/Nebula/blob/a5c718bf9e9ccbafe4ac0ff338799773e80a24ff/README.md#L18)):*
> ```Markdown
> 2. Intrinsic Design prioritizes content
> 3. SEO super good
> 4. Can be exposed as Custom Elements for most flexible usage
> ```

sf


<!-- Sent from Reviewable.io -->

oscl8tr reviewed (commented):
> Reviewed 1 of 1 files at r4, all commit messages.
> *[Reviewable](http://localhost:8080/reviews/Hyperion-Web/Nebula/8)* status: 0 of 1 files reviewed, 3 unresolved discussions (waiting on @earlAchromatic and @prcl8tr)

___
*[`README.md` line 30 at r4](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-Nz--W-TEVVHkzJR9U-b:-Nz--W-TEVVHkzJR9U-c:b32wwo) ([raw file](https://github.com/Hyperion-Web/Nebula/blob/85a834fc01bd48df56e0ddaef9557b3f69cbb107/README.md#L30)):*
> ```Markdown
> ## Interface
>  
> Consumers should import style bundle from the lib and make sure it is inserted before custom styles so that custom styles have the final say. Components (atoms, molecules), compositions, and layouts (which are also `Vue` components) are all available and using `vue-unplugin-components` they can be auto imported.
> ```

sf



_Code quote:_
```Markdown
tions, and layouts (which are also `Vue` components) are all available and using `vue-unplugin-components` they can be auto imported.
```


<!-- Sent from Reviewable.io -->

earlAchromatic reviewed (commented):
> Reviewed 1 of 1 files at r4, all commit messages.
> *[Reviewable](http://localhost:8080/reviews/Hyperion-Web/Nebula/8)* status: all files reviewed, 3 unresolved discussions (waiting on @oscl8tr and @prcl8tr)

___
*[`README.md` line 18 at r4](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-NyvLVU0-FzBiLxWcHGn:-NzEuWCZ1LDkD5-wBb9p:bdk6sbs) ([raw file](https://github.com/Hyperion-Web/Nebula/blob/a5c718bf9e9ccbafe4ac0ff338799773e80a24ff/README.md#L18)):*
<details><summary><i>Previously, earlAchromatic (Jacob Milhorn) wrote…</i></summary><blockquote>

sf
</blockquote></details>

,khjkljk
___
*[`README.md` line 30 at r4](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-Nz--W-TEVVHkzJR9U-b:-NzTw0W72oOAh6Ry3RHN:b-896fix) ([raw file](https://github.com/Hyperion-Web/Nebula/blob/85a834fc01bd48df56e0ddaef9557b3f69cbb107/README.md#L30)):*
<details><summary><i>Previously, oscl8tr wrote…</i></summary><blockquote>

sf
</blockquote></details>

Done.


<!-- Sent from Reviewable.io -->

earlAchromatic commented:
*[`-- commits` line 6 at r4](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-NziI3uQ5wnPE47XsQ7O):*
jkh



_Code quote:_
```
New commits in r1 on 4/7/2023 at 1:35 PM:
- 9093665: Update README.md

New commits in r2 on 7/7/2023 at 2:48 PM:

- 363294b: Update README.md
```


<!-- Sent from Reviewable.io -->

earlAchromatic reviewed (commented):
https://github.com/Reviewable/reviewable-client/pull/425

> Reviewed 1 of 1 files at r4.
> *[Reviewable](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-:-O-DauBRD6MgqCN4qsTu:bihleem)* status: all files reviewed, 4 unresolved discussions (waiting on @oscl8tr and @prcl8tr)

___
*[`-- commits` line 10 at r4](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-NziJmAR9RrYvVtPoCn9):*
sfd



_Code snippet:_
```
sdfasda
```
___
*[`README.md` line 18 at r4](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-NyvLVU0-FzBiLxWcHGn:-O-DT99XCEQViMt4nP7R:bdxa0m4) ([raw file](https://github.com/Hyperion-Web/Nebula/blob/a5c718bf9e9ccbafe4ac0ff338799773e80a24ff/README.md#L18)):*
<details><summary><i>Previously, earlAchromatic (Jacob Milhorn) wrote…</i></summary><blockquote>

,khjkljk
</blockquote></details>

sadfsagaghlj
___
*[`README.md` line 30 at r4](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-Nz--W-TEVVHkzJR9U-b:-NziItUe53aiVO-Cha3i:boyok44) ([raw file](https://github.com/Hyperion-Web/Nebula/blob/85a834fc01bd48df56e0ddaef9557b3f69cbb107/README.md#L30)):*
<details><summary><i>Previously, earlAchromatic (Jacob Milhorn) wrote…</i></summary><blockquote>

Done.
</blockquote></details>

Possibly limit exposed components to assembled molecules and layouts. Abstract away what is in between the content and the complicated layouts and logic.

A workflow for someone using the lib would look like 1. make a page 2. apply a layout 3. fill the layout slots with the appropriate content, be it template for a generated from headless architecture or simply hand written/hardcoded into the slot.
___
*[`README.md` line 30 at r4](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-NziNiTI04mp091mMw2H:-NziNiTJ7rjXM4KlTWaS:b32wp2) ([raw file](https://github.com/Hyperion-Web/Nebula/blob/85a834fc01bd48df56e0ddaef9557b3f69cbb107/README.md#L30)):*
> ```Markdown
> ## Interface
>  
> Consumers should import style bundle from the lib and make sure it is inserted before custom styles so that custom styles have the final say. Components (atoms, molecules), compositions, and layouts (which are also `Vue` components) are all available and using `vue-unplugin-components` they can be auto imported.
> ```

jk



_Code quote:_
```Markdown
olecules), compositions, and layouts (which are also `Vue` components) are all available and using `vue-unplugin-components` they can be auto imported.
```


<!-- Sent from Reviewable.io -->

earlAchromatic reviewed (commented):
asdgsdg

> *[Reviewable](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-:-O-MuEjB8P-YYxIGDmTY:bnnix4s)* status: all files reviewed, 4 unresolved discussions (waiting on @oscl8tr and @prcl8tr)



<!-- Sent from Reviewable.io -->

earlAchromatic commented:
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/> 





<!-- Sent from Reviewable.io -->

earlAchromatic reviewed (commented):
> *[Reviewable](http://localhost:8080/reviews/Hyperion-Web/Nebula/8)* status: all files reviewed, 4 unresolved discussions (waiting on @oscl8tr and @prcl8tr)

___
*[`-- commits` line 8 at r4](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-O-NGakE6P9_ghnCY9RE):*
asg



_Code quote:_
```
- 3a0df44: Update README.md
```
___
*[`README.md` line 30 at r4](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-Nz--W-TEVVHkzJR9U-b:-O-bgha51mGbhA7E1H0M:b-4mxbdl) ([raw file](https://github.com/Hyperion-Web/Nebula/blob/85a834fc01bd48df56e0ddaef9557b3f69cbb107/README.md#L30)):*
> A workflow for someone using the lib would look like 1. make a page 2. apply a layout 3. fill the layout slots with the a


<!-- Sent from Reviewable.io -->

earlAchromatic reviewed (commented):
> *[Reviewable](http://localhost:8080/reviews/Hyperion-Web/Nebula/8)* status: all files reviewed, 4 unresolved discussions (waiting on @oscl8tr and @prcl8tr)

___
*[`-- commits` line 8 at r4](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-O-NGakE6P9_ghnCY9RE):*
<details><summary><i>Previously, earlAchromatic (Jacob Milhorn) wrote…</i></summary><blockquote>

asg
</blockquote></details>

sdfgdsf


<!-- Sent from Reviewable.io -->

earlAchromatic commented:
+bug 





<!-- Sent from Reviewable.io -->

earlAchromatic reviewed (commented):
> *[Reviewable](http://10.0.0.5:8080/reviews/Hyperion-Web/Nebula/8)* status: all files reviewed, 3 unresolved discussions (waiting on @oscl8tr and @prcl8tr)

___
*[`README.md` line 30 at r4](http://10.0.0.5:8080/reviews/Hyperion-Web/Nebula/8#-Nz--W-TEVVHkzJR9U-b:-O-wLC8O-HbRiPWrqEW_:b32wpw) ([raw file](https://github.com/Hyperion-Web/Nebula/blob/85a834fc01bd48df56e0ddaef9557b3f69cbb107/README.md#L30)):*
<details><summary><i>Previously, earlAchromatic (Jacob Milhorn) wrote…</i></summary><blockquote>

> A workflow for someone using the lib would look like 1. make a page 2. apply a layout 3. fill the layout slots with the a
</blockquote></details>

kj


<!-- Sent from Reviewable.io -->

earlAchromatic reviewed (commented):
kj

> *[Reviewable](http://10.0.0.5:8080/reviews/Hyperion-Web/Nebula/8#-:-O-wjW4K8wH8hw5NCgoV:b32wpw)* status: all files reviewed, 3 unresolved discussions (waiting on @oscl8tr and @prcl8tr)



<!-- Sent from Reviewable.io -->

earlAchromatic reviewed (commented):
as

> *[Reviewable](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-:-O0tm76FBT3Fwftq1bAg:b32whj)* status: 0 of 1 files reviewed, 5 unresolved discussions (waiting on @oscl8tr and @prcl8tr)

*[a discussion](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-O0Mt_zp1w2_QJvnQ2oB:-O0Mt_zp1w2_QJvnQ2oC:b-ovser7) (no related file):*
jkhkhl

___
___
*[`README.md` line 22 at r4](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-O0WtPrx5dlHi7D51yNZ:-O-NG_0O5aERyRPIIT2J:b2nic38) ([raw file](https://github.com/Hyperion-Web/Nebula/blob/a5c718bf9e9ccbafe4ac0ff338799773e80a24ff/README.md#L22)):*
> ```Markdown
> 
> 
> A workflow for someone using the lib would look like 1. make a page 2. apply a layout 3. fill the layout slots with the appropriate content, be it template for a generated from headless architecture or simply hand written/hardcoded into the slot.
> ```

sag



_Code quote:_
```Markdown
ould look like 1. make a page 2. apply a layout 3. fill the layout slots with the appropriate content, be it template for a generated from headless architecture or simply hand
```


<!-- Sent from Reviewable.io -->

earlAchromatic reviewed (commented):
lkj

> *[Reviewable](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-:-O0tmTPC38FJH6hUloCm:b2ni752)* status: 0 of 1 files reviewed, 5 unresolved discussions (waiting on @oscl8tr and @prcl8tr)



<!-- Sent from Reviewable.io -->

earlAchromatic reviewed (commented):
kjh

> *[Reviewable](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-:-O0toBtJAD_hLkvEepIH:b2ni6dg)* status: 0 of 1 files reviewed, 5 unresolved discussions (waiting on @oscl8tr and @prcl8tr)



<!-- Sent from Reviewable.io -->

earlAchromatic reviewed (commented):
sdf

> *[Reviewable](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-:-O0tq8R81r4Vw9fDNDB5:b2nic5s)* status: 0 of 1 files reviewed, 5 unresolved discussions (waiting on @oscl8tr and @prcl8tr)



<!-- Sent from Reviewable.io -->

earlAchromatic reviewed (commented):
sadg

> *[Reviewable](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-:-O13c7mi9DtehGu8lRXk:bb7oct2)* status: 0 of 1 files reviewed, 5 unresolved discussions (waiting on @oscl8tr and @prcl8tr)



<!-- Sent from Reviewable.io -->

earlAchromatic reviewed (commented):
> *[Reviewable](http://localhost:8080/reviews/Hyperion-Web/Nebula/8)* status: 0 of 1 files reviewed, 5 unresolved discussions (waiting on @oscl8tr and @prcl8tr)

___
*[`-- commits` line 8 at r4](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-O13wui78bELG0ZBEDUo):*
l;'



_Code quote:_
```
New commits in r2 on 7/7/2023 at 2:48 PM:

- 363294b: Update README.md

- 3a0df44: Update README.md
```
___
*[`README.md` line 26 at r3](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-Ny1rJXz8NlREHcx5d8i:-O13x8pZ-PUsasYb7zPS:buf5qn8) ([raw file](https://github.com/Hyperion-Web/Nebula/blob/85a834fc01bd48df56e0ddaef9557b3f69cbb107/README.md#L26)):*
<details><summary><i>Previously, earlAchromatic (Jacob Milhorn) wrote…</i></summary><blockquote>

sdf
</blockquote></details>

1. All components should be SSR/SSG and template friendly
2. Intrinsic Design prioritizes content
3. SEO super good
4. Can be exposed as Custom Elements for most flexible usage


<!-- Sent from Reviewable.io -->

earlAchromatic commented:
*[`-- commits` line 2 at r4](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-O2XIm-VExAKiTF57F2-):*
agsd



_Code quote:_
```
5: Update README.md
```


<!-- Sent from Reviewable.io -->

earlAchromatic commented:
sdfsd fsdf 





<!-- Sent from Reviewable.io -->

earlAchromatic commented:
*[`-- commits` line 4 at r4](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-O3eFDGN935_4j2Bs3Hc):*
agsdfs


<!-- Sent from Reviewable.io -->

earlAchromatic commented:
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/>





<!-- Sent from Reviewable.io -->

earlAchromatic committed:
Update README.md

earlAchromatic reviewed (commented):
> Reviewed 1 of 1 files at r4.
> *[Reviewable](http://localhost:8080/reviews/Hyperion-Web/Nebula/8)* status: 0 of 1 files reviewed, 58 unresolved discussions (waiting on @oscl8tr and @prcl8tr)

*[a discussion](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-O2XIkvnEmtxyKm9j6cO:-O2XIkvnEmtxyKm9j6cP:bnfp4e4) (no related file):*
<img title="Looks Good to Bunny" alt=":lgtb:" src="/lgtb/11bee6a21d17c673.jpg" height="200"/>

___
*[a discussion](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-O9WMoZZ14xRoARaY5AV:-O9WMoZZ14xRoARaY5AW:bnfp4e4) (no related file):*
<img title="Looks Good to Bunny" alt=":lgtb:" src="/lgtb/f099e01b27cbc5b1.jpg" height="200"/>

___
*[a discussion](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-O9WMqKaB-6hrMtz9Pui:-O9WMqKaB-6hrMtz9Puj:bnfp4e4) (no related file):*
<img title="Looks Good to Bunny" alt=":lgtb:" src="/lgtb/7228c8c08dee581b.jpg" height="200"/>

___
*[a discussion](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-O9WMtnm9HIGvH2rEs5M:-O9WMtnm9HIGvH2rEs5N:bnfp4e4) (no related file):*
<img title="Looks Good to Bunny" alt=":lgtb:" src="/lgtb/15e66b01485179a4.jpg" height="200"/>

___
*[a discussion](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-O9WMwIc7oq4SIddyme4:-O9WMwIc7oq4SIddyme5:bnfp4e4) (no related file):*
<img title="Looks Good to Bunny" alt=":lgtb:" src="/lgtb/bdaab974a510e324.jpg" height="200"/>

___
*[a discussion](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-O9WMyW418T33UGsgOtA:-O9WMyW418T33UGsgOtB:bnfp4e4) (no related file):*
<img title="Looks Good to Bunny" alt=":lgtb:" src="/lgtb/12c319c767903d75.jpg" height="200"/>

___
*[a discussion](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-O9WN-4IA5TyBzQpboZs:-O9WN-4J8fKj1XheDuXm:bnfp4e4) (no related file):*
<img title="Looks Good to Bunny" alt=":lgtb:" src="/lgtb/5d22160289dd315a.jpg" height="200"/>

___
*[a discussion](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-O9WN3Wn44-HprhftFM6:-O9WN3Wn44-HprhftFM7:bnfp4e4) (no related file):*
<img title="Looks Good to Bunny" alt=":lgtb:" src="/lgtb/5d22160289dd315a.jpg" height="200"/>

___
*[a discussion](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-O9WN4kz-z-Pr2GMNKoh:-O9WN4kz-z-Pr2GMNKoi:bnfp4e4) (no related file):*
<img title="Looks Good to Bunny" alt=":lgtb:" src="/lgtb/a35ec528170d2302.jpg" height="200"/>

___
*[a discussion](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-O9WN6EP5qIsgWMnD2EV:-O9WN6EP5qIsgWMnD2EW:bnfp4e4) (no related file):*
<img title="Looks Good to Bunny" alt=":lgtb:" src="/lgtb/756a594c9a2e7824.jpg" height="200"/>

___
*[a discussion](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-O9WN7T82XiIHaFGpOPS:-O9WN7T82XiIHaFGpOPT:bnfp4e4) (no related file):*
<img title="Looks Good to Bunny" alt=":lgtb:" src="/lgtb/6e0490386bb37f66.jpg" height="200"/>

___
*[a discussion](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-O9WN8sp3YUkXsFeyj1t:-O9WN8sp3YUkXsFeyj1u:bnfp4e4) (no related file):*
<img title="Looks Good to Bunny" alt=":lgtb:" src="/lgtb/27d9c89250003b2a.jpg" height="200"/>

___
*[a discussion](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-O9WNBJHDETk2ZyScasJ:-O9WNBJHDETk2ZyScasK:bnfp4e4) (no related file):*
<img title="Looks Good to Bunny" alt=":lgtb:" src="/lgtb/b277fecb0d33b2a9.jpg" height="200"/>

___
*[a discussion](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-O9WNF0F4XNRhupXZwuP:-O9WNF0F4XNRhupXZwuQ:bnfp4e4) (no related file):*
<img title="Looks Good to Bunny" alt=":lgtb:" src="/lgtb/bdaab974a510e324.jpg" height="200"/>

___
*[a discussion](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-O9WNGRV1Q77k34IXBTy:-O9WNGRV1Q77k34IXBTz:bnfp4e4) (no related file):*
<img title="Looks Good to Bunny" alt=":lgtb:" src="/lgtb/f099e01b27cbc5b1.jpg" height="200"/>

___
*[a discussion](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-O9WNHgVAVEZWW4s7lgM:-O9WNHgVAVEZWW4s7lgN:bnfp4e4) (no related file):*
<img title="Looks Good to Bunny" alt=":lgtb:" src="/lgtb/bdaab974a510e324.jpg" height="200"/>

___
*[a discussion](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-O9WNJLTE1axHSVsSzRf:-O9WNJLTE1axHSVsSzRg:bnfp4e4) (no related file):*
<img title="Looks Good to Bunny" alt=":lgtb:" src="/lgtb/b277fecb0d33b2a9.jpg" height="200"/>

___
*[a discussion](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-O9WNMTB6gtjkaouduDa:-O9WNMTC6H9HFE5T7vbI:bnfp4e4) (no related file):*
<img title="Looks Good to Bunny" alt=":lgtb:" src="/lgtb/756a594c9a2e7824.jpg" height="200"/>

___
*[a discussion](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-O9WNNkpAzeHI0_eC0ox:-O9WNNkpAzeHI0_eC0oy:bnfp4e4) (no related file):*
<img title="Looks Good to Bunny" alt=":lgtb:" src="/lgtb/15e66b01485179a4.jpg" height="200"/>

___
___
*[`-- commits` line 3 at r2](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-O2XIlRu2to1plwJyury):*
asdg-- commits
___
*[`-- commits` line 4 at r2](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-O2XIoBN9HKocO2yYqaD):*
sdfag
___
*[`-- commits` line 5 at r4](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-O1qrYkS5CpVJj3t_9jx):*
s
___
*[`-- commits` line 5 at r4](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-O2XInkSDT-e6sWJorPC):*
asdgsdf
___
*[`-- commits` line 6 at r4](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-O1j59R0-XmEuGGQlQ9n):*
sw
___
*[`-- commits` line 6 at r4](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-O3sz5HiEt7-ZQ0iaKPd):*
asdg
___
*[`-- commits` line 6 at r4](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-O3sz7Rn5BfdSIvyHxNS):*
asdgas 
___
*[`-- commits` line 6 at r4](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-O3t2XRu1CdLn_ZMyMjR):*
asdg x
___
*[`-- commits` line 6 at r4](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-O3t2b3fANzaXGSaQkSJ):*
asg 
___
*[`-- commits` line 8 at r4](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-O3sVHUa39YCXR5-AYUq):*
safg 
___
*[`-- commits` line 13 at r4](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-O3sScv-DTBsPQzJI1qb):*
sf
___
*[`-- commits` line 13 at r4](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-O3sSj8r2kxqF16o1Gkl):*
asdf 
___
*[`-- commits` line 13 at r4](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-O3ssbHr5NjvxK1sr6Wy):*
asga 
___
*[`-- commits` line 14 at r4](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-O1U3U-SA-yckAsTcOgg):*
sfasdg 



_Code quote:_
```
- 0e8e619: Update README.md

New commits in r4 on 7/7/2023 at 3:00 PM:
- a5c718b: Update README.md
```
___
*[`-- commits` line 14 at r4](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-O3sSdLd0-QzpEPpNw8H):*
sfasdf
___
*[`-- commits` line 14 at r4](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-O3sXAkk8WYZP5foXKit):*
s
___
*[`README.md` line 15 at r4](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-O3t0wBcAAI1bH7sW957:-O3t0wBcAAI1bH7sW958:bb7cx9e) ([raw file](https://github.com/Hyperion-Web/Nebula/blob/a5c718bf9e9ccbafe4ac0ff338799773e80a24ff/README.md#L15)):*
> ```Markdown
> ## Design Objectives
> 
> 1. All components should be SSR/SSG and template friendly
> ```

adf 
___
*[`README.md` line 15 at r4](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-O3t13qSEcTlnxoIKGpB:-O3t13qSEcTlnxoIKGpC:b32wwo) ([raw file](https://github.com/Hyperion-Web/Nebula/blob/a5c718bf9e9ccbafe4ac0ff338799773e80a24ff/README.md#L15)):*
> ```Markdown
> ## Design Objectives
> 
> 1. All components should be SSR/SSG and template friendly
> ```

sf
___
*[`README.md` line 15 at r4](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-O3t15X9D2mBD3Z1iwXf:-O3t15X9D2mBD3Z1iwXg:b2nic5t) ([raw file](https://github.com/Hyperion-Web/Nebula/blob/a5c718bf9e9ccbafe4ac0ff338799773e80a24ff/README.md#L15)):*
> ```Markdown
> ## Design Objectives
> 
> 1. All components should be SSR/SSG and template friendly
> ```

sdg
___
*[`README.md` line 15 at r4](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-O3t16EjE0PX6-nRn9pH:-O3t16EjE0PX6-nRn9pI:b-7t61rt) ([raw file](https://github.com/Hyperion-Web/Nebula/blob/a5c718bf9e9ccbafe4ac0ff338799773e80a24ff/README.md#L15)):*
> ```Markdown
> ## Design Objectives
> 
> 1. All components should be SSR/SSG and template friendly
> ```

asdf 
___
*[`README.md` line 15 at r4](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-O3t16r21alQ4cKbF-qP:-O3t16r21alQ4cKbF-qQ:b-t0vbam) ([raw file](https://github.com/Hyperion-Web/Nebula/blob/a5c718bf9e9ccbafe4ac0ff338799773e80a24ff/README.md#L15)):*
> ```Markdown
> ## Design Objectives
> 
> 1. All components should be SSR/SSG and template friendly
> ```

asdfg 
___
*[`README.md` line 15 at r4](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-O3t19DsEzNYHwM2D-B5:-O3t19DsEzNYHwM2D-B6:b-7t61rt) ([raw file](https://github.com/Hyperion-Web/Nebula/blob/a5c718bf9e9ccbafe4ac0ff338799773e80a24ff/README.md#L15)):*
> ```Markdown
> ## Design Objectives
> 
> 1. All components should be SSR/SSG and template friendly
> ```

asdf 
___
*[`README.md` line 15 at r4](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-O3t1D1j8-BjhYpVfTjO:-O3t1D1j8-BjhYpVfTjP:b-7t61qy) ([raw file](https://github.com/Hyperion-Web/Nebula/blob/a5c718bf9e9ccbafe4ac0ff338799773e80a24ff/README.md#L15)):*
> ```Markdown
> ## Design Objectives
> 
> 1. All components should be SSR/SSG and template friendly
> ```

asdg 
___
*[`README.md` line 15 at r4](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-O3t1EAC3XC18E3Mo1TQ:-O3t1EAC3XC18E3Mo1TR:bmyf5l0) ([raw file](https://github.com/Hyperion-Web/Nebula/blob/a5c718bf9e9ccbafe4ac0ff338799773e80a24ff/README.md#L15)):*
> ```Markdown
> ## Design Objectives
> 
> 1. All components should be SSR/SSG and template friendly
> ```

adsfdsf 
___
*[`README.md` line 15 at r4](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-O3t1G20-Ue19KW7c40q:-O3t1G20-Ue19KW7c40r:b-7t61qy) ([raw file](https://github.com/Hyperion-Web/Nebula/blob/a5c718bf9e9ccbafe4ac0ff338799773e80a24ff/README.md#L15)):*
> ```Markdown
> ## Design Objectives
> 
> 1. All components should be SSR/SSG and template friendly
> ```

asdg 
___
*[`README.md` line 15 at r4](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-O902dYg8KDXn1ye5cY-:-O902dYhAJPsDemWkFo9:b-7jja1c) ([raw file](https://github.com/Hyperion-Web/Nebula/blob/a5c718bf9e9ccbafe4ac0ff338799773e80a24ff/README.md#L15)):*
> ```Markdown
> ## Design Objectives
> 
> 1. All components should be SSR/SSG and template friendly
> ```

sdfss



_Code quote:_
```Markdown
## Design Objectives

1. All components should be SSR/SSG and template friendly
```
___
*[`README.md` line 16 at r4](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-O3t0uMT6oUC0XYP-hJA:-O3t0uMT6oUC0XYP-hJB:b-7t61rt) ([raw file](https://github.com/Hyperion-Web/Nebula/blob/a5c718bf9e9ccbafe4ac0ff338799773e80a24ff/README.md#L16)):*
> ```Markdown
> 
> 1. All components should be SSR/SSG and template friendly
> 2. Intrinsic Design prioritizes content
> ```

asdf 
___
*[`README.md` line 16 at r4](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-O3t0v3XEPrNFAWDh5cz:-O3t0v3XEPrNFAWDh5d-:b-7tfbgb) ([raw file](https://github.com/Hyperion-Web/Nebula/blob/a5c718bf9e9ccbafe4ac0ff338799773e80a24ff/README.md#L16)):*
> ```Markdown
> 
> 1. All components should be SSR/SSG and template friendly
> 2. Intrinsic Design prioritizes content
> ```

adsf 
___
*[`README.md` line 17 at r4](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-O3suex83CyYSQie9BFp:-O-MuBiQ1Jouk5Ng6iYN:bb7oct2) ([raw file](https://github.com/Hyperion-Web/Nebula/blob/a5c718bf9e9ccbafe4ac0ff338799773e80a24ff/README.md#L17)):*
> ```Markdown
> 1. All components should be SSR/SSG and template friendly
> 2. Intrinsic Design prioritizes content
> 3. SEO super good
> ```

sadg



_Code quote:_
```Markdown
1. All components should be SSR/SSG and template friendly
2. Intrinsic Design prioritizes content
3. SEO super good
```
___
*[`README.md` line 22 at r4](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-O1tXevF6u7Z-oqVEAPJ:-O1tXevF6u7Z-oqVEAPK:b-7jj0pi) ([raw file](https://github.com/Hyperion-Web/Nebula/blob/a5c718bf9e9ccbafe4ac0ff338799773e80a24ff/README.md#L22)):*
> ```Markdown
> 
> 
> A workflow for someone using the lib would look like 1. make a page 2. apply a layout 3. fill the layout slots with the appropriate content, be it template for a generated from headless architecture or simply hand written/hardcoded into the slot.
> ```

sdsfs
___
*[`README.md` line 22 at r4](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-O2XIqpb615L3FUbl5B5:-O2XIqpb615L3FUbl5B6:bb7czuk) ([raw file](https://github.com/Hyperion-Web/Nebula/blob/a5c718bf9e9ccbafe4ac0ff338799773e80a24ff/README.md#L22)):*
> ```Markdown
> 
> 
> A workflow for someone using the lib would look like 1. make a page 2. apply a layout 3. fill the layout slots with the appropriate content, be it template for a generated from headless architecture or simply hand written/hardcoded into the slot.
> ```

agsd
___
*[`README.md` line 22 at r4](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-O3sZ9Mb9LWaH3xB2Xfl:-O3sZ9Mc-pcZwZdlHXse:b32wtc) ([raw file](https://github.com/Hyperion-Web/Nebula/blob/a5c718bf9e9ccbafe4ac0ff338799773e80a24ff/README.md#L22)):*
> ```Markdown
> 
> 
> A workflow for someone using the lib would look like 1. make a page 2. apply a layout 3. fill the layout slots with the appropriate content, be it template for a generated from headless architecture or simply hand written/hardcoded into the slot.
> ```

oj
___
*[`README.md` line 22 at r4](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-O3st9Xa0T3cu_hotvFt:-O3st9Xa0T3cu_hotvFu:bb7of0w) ([raw file](https://github.com/Hyperion-Web/Nebula/blob/a5c718bf9e9ccbafe4ac0ff338799773e80a24ff/README.md#L22)):*
> ```Markdown
> 
> 
> A workflow for someone using the lib would look like 1. make a page 2. apply a layout 3. fill the layout slots with the appropriate content, be it template for a generated from headless architecture or simply hand written/hardcoded into the slot.
> ```

sdf 


<!-- Sent from Reviewable.io -->

oscl8tr commented:
:lgtb:

earlAchromatic committed:
Merge branch 'master' into earlAchromatic-patch-5

earlAchromatic reviewed (commented):
<img title="Looks Good to Bunny" alt=":lgtb:" src="https://reviewable.io/lgtb/6e0490386bb37f66.jpg" height="200"/>asdf 

> *[Reviewable](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-:-O4GqFSu7gkJpe3eF1kp:b-774sr4)* status: 0 of 1 files reviewed, 57 unresolved discussions (waiting on @oscl8tr and @prcl8tr)

*[a discussion](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-O9WMoZZ14xRoARaY5AV:-O9WP7f52a-SprsmMQme:bnfp4e4) (no related file):*
<details><summary><i>Previously, earlAchromatic (Jacob Milhorn) wrote…</i></summary><blockquote>

<img title="Looks Good to Bunny" alt=":lgtb:" src="https://reviewable.io/lgtb/f099e01b27cbc5b1.jpg" height="200"/>
</blockquote></details>

<img title="Looks Good to Bunny" alt=":lgtb:" src="https://reviewable.io/lgtb/12c319c767903d75.jpg" height="200"/>

___
*[a discussion](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-O9WN8sp3YUkXsFeyj1t:-O9zx1vkB7fpWdafqJRC:b1oj8dg) (no related file):*
<details><summary><i>Previously, earlAchromatic (Jacob Milhorn) wrote…</i></summary><blockquote>

<img title="Looks Good to Bunny" alt=":lgtb:" src="https://reviewable.io/lgtb/27d9c89250003b2a.jpg" height="200"/>
</blockquote></details>

asdf dsf sdf adsf s

___
___
*[`-- commits` line 6 at r4](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-O3t2XRu1CdLn_ZMyMjR):*
<details><summary><i>Previously, earlAchromatic (Jacob Milhorn) wrote…</i></summary><blockquote>

asdg x
</blockquote></details>

asdf asdf 


<!-- Sent from Reviewable.io -->

oscl8tr reviewed (changes requested):
> Reviewed 1 of 1 files at r6, all commit messages.
> *[Reviewable](http://localhost:8080/reviews/Hyperion-Web/Nebula/8)* status: 0 of 1 files reviewed, 58 unresolved discussions (waiting on @earlAchromatic and @prcl8tr)

___
*[`README.md` line 18 at r1](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-O0QEW4_-NKbLQT1y3xS:-O0QEW4_-NKbLQT1y3xT:b-t0tfh5) ([raw file](https://github.com/Hyperion-Web/Nebula/blob/9093665033e6f5ea05f979d0a24a5b4441430575/README.md#L18)):*
> ```Markdown
> 2. Intrinsic Design prioritizes content
> 3. SEO super good
> 4. Can be exposed as Custom Elements for most flexible usage
> ```

asgdsf



_Code quote:_
```Markdown
4. Can be exposed as Custom Elements for most flexible usage
```


<!-- Sent from Reviewable.io -->

earlAchromatic reviewed (commented):
> Reviewed 1 of 1 files at r6, all commit messages.
> *[Reviewable](http://localhost:8080/reviews/Hyperion-Web/Nebula/8)* status: all files reviewed, 58 unresolved discussions (waiting on @prcl8tr)



<!-- Sent from Reviewable.io -->

earlAchromatic commented:
how about `#FFA22E`

`rgba(127, 255, 0, 0.5)`





<!-- Sent from Reviewable.io -->

earlAchromatic reviewed (commented):
> *[Reviewable](http://localhost:8080/reviews/Hyperion-Web/Nebula/8)* status: all files reviewed, 58 unresolved discussions (waiting on @oscl8tr and @prcl8tr)

___
*[`README.md` line 22 at r4](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-O3st9Xa0T3cu_hotvFt:-OK4UwJjDJZ1VCiwSU12:bb7cxmk) ([raw file](https://github.com/Hyperion-Web/Nebula/blob/a5c718bf9e9ccbafe4ac0ff338799773e80a24ff/README.md#L22)):*
<details><summary><i>Previously, earlAchromatic (Jacob Milhorn) wrote…</i></summary><blockquote>

sdf
</blockquote></details>

adsg


<!-- Sent from Reviewable.io -->

earlAchromatic reviewed (commented):
asdg

> *[Reviewable](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-:-OK4Vqxd4GaZqQxOtLxg:bb7d8e2)* status: all files reviewed, 58 unresolved discussions (waiting on @oscl8tr and @prcl8tr)



<!-- Sent from Reviewable.io -->

earlAchromatic reviewed (commented):
asgssd

> *[Reviewable](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-:-OK4W6eDAgFIlC9jaQvg:b-t0t4cs)* status: all files reviewed, 58 unresolved discussions (waiting on @oscl8tr and @prcl8tr)



<!-- Sent from Reviewable.io -->

earlAchromatic reviewed (commented):
asdgsadf 

> *[Reviewable](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-:-OK4WIknAal5WZyDqiVV:b13yzpy)* status: all files reviewed, 58 unresolved discussions (waiting on @oscl8tr and @prcl8tr)



<!-- Sent from Reviewable.io -->

earlAchromatic reviewed (commented):
asdfsdf 

> *[Reviewable](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-:-OK4X2nb40hmKvB85eHN:bmxm6ro)* status: all files reviewed, 58 unresolved discussions (waiting on @oscl8tr and @prcl8tr)



<!-- Sent from Reviewable.io -->

earlAchromatic reviewed (commented):
> *[Reviewable](http://localhost:8080/reviews/Hyperion-Web/Nebula/8)* status: all files reviewed, 52 unresolved discussions (waiting on @oscl8tr and @prcl8tr)

___
*[`README.md` line 22 at r4](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-O2XIqpb615L3FUbl5B5:-OK4hIJmA1xEWMZLptA2:b-wm7xfn) ([raw file](https://github.com/Hyperion-Web/Nebula/blob/a5c718bf9e9ccbafe4ac0ff338799773e80a24ff/README.md#L22)):*
<details><summary><i>Previously, earlAchromatic (Jacob Milhorn) wrote…</i></summary><blockquote>

agsd
</blockquote></details>

If you open the guide overlay (that marks hot areas to click on) while a multiselect dropdown is open, then clicking on a guide area appears to close then reopen the dropdown. It looks confusing, and if the clicked guide is in the dropdown it sometimes messes up the highlighting. Strangely, further clicks are fine.

The guide overlay stops `mousedown` events (see `overlay.html`), and the guide implementation used to unbind/rebind the intent of the SemanticUI dropdown widget (see `guide.js`, search for `unbind intent` and `bind intent`), but this doesn't appear to have been ported when we switched to `multiselect`.
___
*[`-- commits` line 16 at r6](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-OK4gbs9BxcyfZvuaa1v:-OK4gbs9BxcyfZvuaa1w:bb7of0w):*
sdf 
___
*[`README.md` line 19 at r6](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-OK4hMs68IfZfyWCtwqr:-OK4hMs68IfZfyWCtwqs:ba552l6) ([raw file](https://github.com/Hyperion-Web/Nebula/blob/84c6dae8e6978dd1833e4f879ecd6b84be028075/README.md#L19)):*
> ```Markdown
> 4. Can be exposed as Custom Elements for most flexible usage
> 
>             ┌─────────────────────────┐
> ```

I can imagine an alternative workflow where you quickly scan all of the drafts you are about to send before publishing and if there is a draft that blocks or is otherwise affecting the review in a way you do not intend, clicking the draft in the dropdown takes you directly to the draft in the diffs so proper edits can be made.

Clicking the draft in the preview already takes you to the right place in the review, but the message previews don’t contain as much information as they could. This is because that list of drafts in the message preview was originally modeled to resemble the batched message that goes out over GitHub, which doesn’t co



_Code quote:_
```Markdown
4. Can be exposed as Custom Elements for most flexible usage

            ┌─────────────────────────┐
```
___
*[`README.md` line 20 at r6](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-OK4_p9B0SjP4xxhp8wM:-OK4_p9B0SjP4xxhp8wN:brk795y) ([raw file](https://github.com/Hyperion-Web/Nebula/blob/84c6dae8e6978dd1833e4f879ecd6b84be028075/README.md#L20)):*
> ```Markdown
> 
>             ┌─────────────────────────┐
>             ┌─┐┌─┐┌─┐┌─┐┌─┐┌─┐┌─┐┌─┐┌─┐  nav/approach
> ```

FYI use this
___
*[`README.md` line 27 at r6](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-OK4auhw-IfvKid8rd54:-OK4auhw-IfvKid8rd55:b2ni1ea) ([raw file](https://github.com/Hyperion-Web/Nebula/blob/84c6dae8e6978dd1833e4f879ecd6b84be028075/README.md#L27)):*
> ```Markdown
> 
> 
> ### WIP
> ```

dsf


<!-- Sent from Reviewable.io -->

earlAchromatic commented:
*[`README.md` line 30 at r6](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-OKU4fnW7lTj4nMOnUlR:-OKU4fnW7lTj4nMOnUlS:bfxeltz) ([raw file](https://github.com/Hyperion-Web/Nebula/blob/84c6dae8e6978dd1833e4f879ecd6b84be028075/README.md#L30)):*
> ```Markdown
> 
> - default unocss generated utility classes - how to let user override? UNO CSS CLI as automation?
>   - need to expose our config as part of the package then... 
> ```

Whatever.


<!-- Sent from Reviewable.io -->

oscl8tr commented:
*[`README.md` line 30 at r6](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-OKU4fnW7lTj4nMOnUlR:-OKU4lFv0jAI3Sl8ZRDy:b-n8gt98) ([raw file](https://github.com/Hyperion-Web/Nebula/blob/84c6dae8e6978dd1833e4f879ecd6b84be028075/README.md#L30)):*
<details><summary><i>Previously, earlAchromatic (Jacob Milhorn) wrote…</i></summary><blockquote>

Whatever.
</blockquote></details>

Well, really now.


<!-- Sent from Reviewable.io -->

earlAchromatic commented:
*[`README.md` line 30 at r6](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-OKU4fnW7lTj4nMOnUlR:-OKU4q2GBxWSCbtNOqYH:b-x7bapr) ([raw file](https://github.com/Hyperion-Web/Nebula/blob/84c6dae8e6978dd1833e4f879ecd6b84be028075/README.md#L30)):*
<details><summary><i>Previously, oscl8tr wrote…</i></summary><blockquote>

Well, really now.
</blockquote></details>

I guess so


<!-- Sent from Reviewable.io -->

earlAchromatic commented:
*[`README.md` line 30 at r6](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-OKU4fnW7lTj4nMOnUlR:-OKU4v316xNLHwiAlKRN:b-jrr1h0) ([raw file](https://github.com/Hyperion-Web/Nebula/blob/84c6dae8e6978dd1833e4f879ecd6b84be028075/README.md#L30)):*
<details><summary><i>Previously, earlAchromatic (Jacob Milhorn) wrote…</i></summary><blockquote>

I guess so
</blockquote></details>

But really, is it


<!-- Sent from Reviewable.io -->

earlAchromatic commented:
*[`README.md` line 30 at r6](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-OKU4fnW7lTj4nMOnUlR:-OKU52ai3KNotPgYqKQq:b-7t61rt) ([raw file](https://github.com/Hyperion-Web/Nebula/blob/84c6dae8e6978dd1833e4f879ecd6b84be028075/README.md#L30)):*
<details><summary><i>Previously, earlAchromatic (Jacob Milhorn) wrote…</i></summary><blockquote>

But really, is it
</blockquote></details>

asdf 


<!-- Sent from Reviewable.io -->

earlAchromatic commented:
*[`README.md` line 30 at r6](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-OKU4fnW7lTj4nMOnUlR:-OKU5qJ27xnmBXp4ZnpD:bb6x75v) ([raw file](https://github.com/Hyperion-Web/Nebula/blob/84c6dae8e6978dd1833e4f879ecd6b84be028075/README.md#L30)):*
<details><summary><i>Previously, earlAchromatic (Jacob Milhorn) wrote…</i></summary><blockquote>

asdf
</blockquote></details>

How.


<!-- Sent from Reviewable.io -->

oscl8tr commented:
*[`README.md` line 30 at r6](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-OKU4fnW7lTj4nMOnUlR:-OKU5siUC1d0xzeEjofx:bb7qzwz) ([raw file](https://github.com/Hyperion-Web/Nebula/blob/84c6dae8e6978dd1833e4f879ecd6b84be028075/README.md#L30)):*
<details><summary><i>Previously, earlAchromatic (Jacob Milhorn) wrote…</i></summary><blockquote>

How.
</blockquote></details>

well


<!-- Sent from Reviewable.io -->

earlAchromatic commented:
*[`README.md` line 30 at r6](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-OKU4fnW7lTj4nMOnUlR:-OKU5wfV3JLhSdO2X5pf:b2ni4eu) ([raw file](https://github.com/Hyperion-Web/Nebula/blob/84c6dae8e6978dd1833e4f879ecd6b84be028075/README.md#L30)):*
<details><summary><i>Previously, oscl8tr wrote…</i></summary><blockquote>

well
</blockquote></details>

huh


<!-- Sent from Reviewable.io -->

oscl8tr commented:
*[`README.md` line 30 at r6](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-OKU4fnW7lTj4nMOnUlR:-OKU6-3B8IIduJ5AtdIW:b-rnr90d) ([raw file](https://github.com/Hyperion-Web/Nebula/blob/84c6dae8e6978dd1833e4f879ecd6b84be028075/README.md#L30)):*
<details><summary><i>Previously, earlAchromatic (Jacob Milhorn) wrote…</i></summary><blockquote>

huh
</blockquote></details>

doneso


<!-- Sent from Reviewable.io -->

earlAchromatic commented:
+reviewer:@oscl8tr 





<!-- Sent from Reviewable.io -->

earlAchromatic reviewed (commented):
dsf 

/botcomnd

> Dismissed @oscl8tr from a discussion.
> *[Reviewable](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-:-OKc512pCTY5__iV_RU2:bb7f5ce,-OKbqKQP3IYUYcKl4EMX:b-yfk9ly)* status: all files reviewed, 52 unresolved discussions (waiting on @oscl8tr)

___
*[`README.md` line 15 at r4](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-O3t0wBcAAI1bH7sW957:-OKC9sZi5koNlRH5J83j:b-z0flec) ([raw file](https://github.com/Hyperion-Web/Nebula/blob/a5c718bf9e9ccbafe4ac0ff338799773e80a24ff/README.md#L15)):*
<details><summary><i>Previously, earlAchromatic (Jacob Milhorn) wrote…</i></summary><blockquote>

adf
</blockquote></details>

http://localhost:8080/reviews/Hyperion-Web/Nebula/8#file-NSRnEWcwxF7xXXhlzvE
___
*[`README.md` line 22 at r4](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-O3sZ9Mb9LWaH3xB2Xfl:-OKSJiZw8N3ZK8FJOuxA:b3ksu) ([raw file](https://github.com/Hyperion-Web/Nebula/blob/a5c718bf9e9ccbafe4ac0ff338799773e80a24ff/README.md#L22)):*
<details><summary><i>Previously, earlAchromatic (Jacob Milhorn) wrote…</i></summary><blockquote>

oj
</blockquote></details>

s



_Suggestion:_
```Markdown
A workflow for someone using the lib would look like1. make a page 2. apply a layout 3. fill the layout slots with the appropriate content, be it template for a generated from headless architecture or simply hand written/hardcoded into the slot.
```
___
*[`README.md` line 27 at r6](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-OK4auhw-IfvKid8rd54:-OKSJejo9bC-xEBvs6aR:bb7of2u) ([raw file](https://github.com/Hyperion-Web/Nebula/blob/84c6dae8e6978dd1833e4f879ecd6b84be028075/README.md#L27)):*
<details><summary><i>Previously, earlAchromatic (Jacob Milhorn) wrote…</i></summary><blockquote>

dsf
</blockquote></details>

sdff



_Suggestion (i):_
```Markdown
### WIP sdfs  
```



_Code snippet (ii):_
```Markdown
asdf 
```
___
*[`-- commits` line 3 at r6](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-OKCeODp1jFPykZKgcC3:-OKCeODp1jFPykZKgcC4:b2ni43n):*
hhh


<!-- Sent from Reviewable.io -->

earlAchromatic reviewed (commented):
Here are my general thoughts.
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/>

> *[Reviewable](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-:-OKcLzrj2QZC0AfruVTN:b-4mku2p)* status: 0 of 1 files reviewed, 52 unresolved discussions (waiting on @oscl8tr)

___
*[`README.md` line 17 at r4](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-O3suex83CyYSQie9BFp:-OKcLTYJ5_S8S18NDmJQ:bveufvd) ([raw file](https://github.com/Hyperion-Web/Nebula/blob/a5c718bf9e9ccbafe4ac0ff338799773e80a24ff/README.md#L17)):*
<details><summary><i>Previously, earlAchromatic (Jacob Milhorn) wrote…</i></summary><blockquote>

sadg
</blockquote></details>

Well, what about this?
___
*[`README.md` line 12 at r6](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-OKcLVag6WbdqRA2_AlH:-OKcLVag6WbdqRA2_AlI:bvggc83) ([raw file](https://github.com/Hyperion-Web/Nebula/blob/84c6dae8e6978dd1833e4f879ecd6b84be028075/README.md#L12)):*
> ```Markdown
> ecture. 
> 
> ## Design Objectives
> ```

Is this workable?
___
*[`README.md` line 34 at r6](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-OKcLGNp3fpeDJZXUmnn:-OKbyb4R1cave63M-Wzq:bnfp4e4) ([raw file](https://github.com/Hyperion-Web/Nebula/blob/84c6dae8e6978dd1833e4f879ecd6b84be028075/README.md#L34)):*
> ```Markdown
> ### Opinions on Island Architecture frameworks
> 
> - eleventy's islands don't give flexibility for nested components without a bit of custom configuration. 
> ```

<img title="Looks Good to Bunny" alt=":lgtb:" src="https://reviewable.io/lgtb/e7e49f527e2194dd.jpg" height="200"/>


<!-- Sent from Reviewable.io -->

earlAchromatic reviewed (commented):
what aobut i.

> *[Reviewable](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-:-OL_mpNr-bHWg6r2rHDm:bfnyjd5)* status: 0 of 1 files reviewed, 52 unresolved discussions (waiting on @oscl8tr)

___
*[a discussion](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-O2XIkvnEmtxyKm9j6cO:-OL_mjNi-62JStUv8TtW:bg6a3xy) (no related file):*
<details><summary><i>Previously, earlAchromatic (Jacob Milhorn) wrote…</i></summary><blockquote>

<img title="Looks Good to Bunny" alt=":lgtb:" src="https://reviewable.io/lgtb/11bee6a21d17c673.jpg" height="200"/>
</blockquote></details>

lorem ipsum
___
*[`-- commits` line 6 at r4](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-O3t2XRu1CdLn_ZMyMjR:-OL_mk_xDmZswYI-Wsp-:bb7oqit):*
<details><summary><i>Previously, earlAchromatic (Jacob Milhorn) wrote…</i></summary><blockquote>

asdf asdf
</blockquote></details>

ssss
___
*[`README.md` line 30 at r4](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-Nz--W-TEVVHkzJR9U-b:-OL_hrHnDY0gY93l0JyJ:b7aulet) ([raw file](https://github.com/Hyperion-Web/Nebula/blob/85a834fc01bd48df56e0ddaef9557b3f69cbb107/README.md#L30)):*
<details><summary><i>Previously, earlAchromatic (Jacob Milhorn) wrote…</i></summary><blockquote>

kj
</blockquote></details>

asdfasdfadsfadsf 

asdf 
___
*[`README.md` line 40 at r6](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-OL_mm540Nswvmdr9Zfg:-OL_mm540Nswvmdr9Zfh:b32wwm) ([raw file](https://github.com/Hyperion-Web/Nebula/blob/84c6dae8e6978dd1833e4f879ecd6b84be028075/README.md#L40)):*
> ```Markdown
> - Astro
> 
> - https://github.com/teambit/bit
> ```

sd



_Code quote:_
```Markdown
### Opinions on Island Architecture frameworks

- eleventy's islands don't give flexibility for nested components without a bit of custom configuration.

- Some strange vite voodoo is breaking iles in my sandbox because it is trying to resolve urls inappropriately. I think.

- Astro

- https://github.com/teambit/bit
```


<!-- Sent from Reviewable.io -->

earlAchromatic commented:
*[`README.md` line 27 at r6](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-OK4auhw-IfvKid8rd54:-OQkguvc6vHltl7lTsJz:b-34zkey) ([raw file](https://github.com/Hyperion-Web/Nebula/blob/84c6dae8e6978dd1833e4f879ecd6b84be028075/README.md#L27)):*
<details><summary><i>Previously, earlAchromatic (Jacob Milhorn) wrote…</i></summary><blockquote>

sdff
</blockquote></details>

+reviewer:@prcl8tr 


<!-- Sent from Reviewable.io -->

earlAchromatic commented:
> | Name | Age | Occupation |
> | --- | --- | --- |
> | Alice | 30 | Engineer |
> | Bob | 25 | Designer |
> | Charlie | 35 | Product Manager |


<!-- Sent from Reviewable.io -->

earlAchromatic commented:
+assignee:@prcl8tr 





<!-- Sent from Reviewable.io -->

earlAchromatic commented:
<details><summary><i>Previously, earlAchromatic (Jacob Milhorn) wrote…</i></summary><blockquote>

<img title="Looks Good to Bunny" alt=":lgtb:" src="https://reviewable.io/lgtb/fd239d01759e50b5.jpg" height="200"/>
</blockquote></details>

+@prcl8tr 


<!-- Sent from Reviewable.io -->

earlAchromatic committed:
Remove comment on Vite issues in README

Removed a comment about Vite issues in the context of Island Architecture frameworks.

earlAchromatic committed:
Remove 'View on GitHub' link from index.md

Removed the 'View on GitHub' link from the index.

earlAchromatic committed:
Add GitHub link to the index page

earlAchromatic committed:
Remove GitHub link from index.md

Removed 'View on GitHub' link from the index.

earlAchromatic committed:
Add GitHub link to index.md

earlAchromatic committed:
Fix formatting issue in index.md

earlAchromatic commented:
*[`-- commits` line 26 at r8](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-Og3O_-w6jNynO1Yl0-d:-Og3O_-w6jNynO1Yl0-e:b-t0vc07):*
asdefg


<!-- Sent from Reviewable.io -->

earlAchromatic commented:
*[`-- commits` line 13 at r4](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-O3sScv-DTBsPQzJI1qb:-OVxumY_5wppYHZo_Mnr:bqlvinm):*
<details><summary><i>Previously, earlAchromatic (Jacob Milhorn) wrote…</i></summary><blockquote>

sf
</blockquote></details>

+assignee:@Riley0000 


<!-- Sent from Reviewable.io -->

earlAchromatic commented:
*[`-- commits` line 11 at r8](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-Og3RC6c-IfVMChOHg-F:-Og3RC6c-IfVMChOHg-G:b32wwo):*
sf


<!-- Sent from Reviewable.io -->

earlAchromatic commented:
*[`-- commits` line 1 at r8](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-Og3RH0o4fSLQe2O0y-Y:-Og3RH0o4fSLQe2O0y-Z:bt4ks04):*
agsdg

asdf
dfs


<!-- Sent from Reviewable.io -->

earlAchromatic commented:
*[`-- commits` line 4 at r4](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-O3eFDGN935_4j2Bs3Hc:-Og3SD1B2QMP6z-RipTK:b-janb9):*
<details><summary><i>Previously, earlAchromatic (Jacob Milhorn) wrote…</i></summary><blockquote>

agsdfs
</blockquote></details>

asdg

sdf


<!-- Sent from Reviewable.io -->

earlAchromatic commented:
*[`README.md` line 27 at r8](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-Og3SIMl4vf16D35F7zx:-Og3SIMl4vf16D35F7zy:b32wh7) ([raw file](https://github.com/Hyperion-Web/Nebula/blob/01aca471f624e1ab1013ac9ea9a761e1f78ad019/README.md#L27)):*
> ```Markdown
> 
> 
> ### WIP
> ```

ag


<!-- Sent from Reviewable.io -->

earlAchromatic reviewed (commented):
fofofofofofofoofofofoofofofofofofofofofofofofofofofofofofffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff

> @earlAchromatic reviewed all commit messages, made 2 comments, resolved 4 discussions, and dismissed @oscl8tr from a discussion.
> *[Reviewable](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-:-OQubLxKAFJNa9b9m19e:b-761u1o)* status: 0 of 2 files reviewed, 52 unresolved discussions (waiting on @oscl8tr and @prcl8tr).

___
*[a discussion](http://localhost:8080/reviews/Hyperion-Web/Nebula/8#-O9WMoZZ14xRoARaY5AV:-OVxuq_yAaJA-k5_bkS2:b3kqu) (no related file):*
<details><summary><i>Previously, earlAchromatic (Jacob Milhorn) wrote…</i></summary><blockquote>

<img title="Looks Good to Bunny" alt=":lgtb:" src="https://reviewable.io/lgtb/b277fecb0d33b2a9.jpg" height="200"/>
</blockquote></details>

+


<!-- Sent from Reviewable.io -->
